rebar.config

```diff
@@ -8,7 +8,7 @@
     {platform_define, "^[2-9]", 'POST_OTP_18'}
 ]}.
 
-{project_plugins, [covertool]}.
+{project_plugins, [covertool, {rebar3_hank, "~> 0.3.0"}]}.
```

Comment:
Hm... while it's acceptable, I'd refrain from using `~>` prior to 1.0.0. The reason is in semver's specification: https://semver.org/#spec-item-4

Reply:
Not sure I follow on this one? Technically rebar supports `~>`, iirc.

Reply:
Sure. Let me explain further.

Right now, `rebar3_hank` is 0.3.0.
`rebar3` doesn't **lock** versions for plugins as it does for dep.s.
Let's now assume `rebar3_hank` 0.4.0 comes out with a breaking interface (it can, as per semver).
He/she who pulls in `rebar3_hex` from GitHub and runs the plugin might see unexpected breaking changes.

I was (implicitly) proposing that `0.3.0` is used, not `~> 0.3.0`, before `1.0.0` for the aforementioned reasons.

Reply:
Ahh I see. Happy to accept a PR 😁

Reply:
"It's acceptable" 😄